src/Pages/SingleVideo/SingleVideo.jsx

```diff
@@ -0,0 +1,88 @@
+import "./SingleVideo.css";
+import { useState } from "react";
+import { useVideo, useAuth, useNotes } from "../../Hooks";
+import { Loading, Sidebar, PlaylistModal, NoteInput, NoteList } from "../../Components";
+import { useParams, useNavigate } from "react-router-dom";
+import ReactPlayer from "react-player";
+import { getCreatorImg, getVideoUrl } from "../../Utils";
+
+export default function SingleVideo() {
+    const [showPlaylistModal, setPlaylistModal] = useState(false);
+    const navigate = useNavigate();
+
+    const {
+        videoState: { videoList, isLoading },
+    } = useVideo();
+    const {
+        authState: { token },
+    } = useAuth();
+
+    const { addNoteHandler } = useNotes();
+
+    const { videoId } = useParams();
+    const video = videoList?.find((each) => each._id === videoId);
+
+    function noteSubmitHandler({ e, note, setNoteInput }) {
+        e.preventDefault();
+        if (note.title.trim() || note.text.trim()) {
+            addNoteHandler({ token, note: { ...note, videoId } });
+            setNoteInput({ title: "", text: "" });
+        }
+    }
+
+    return (
+        <div className="main-wrapper">
+            <Sidebar />
+            {isLoading ? (
+                <Loading />
+            ) : (
+                <main className="main pt-12 single-video-wrapper">
+                    <div className="single-video">
+                        <ReactPlayer width="100%" height="30rem" url={getVideoUrl(videoId)} />
+
```

Comment:
Suggestion: set Controls property to true , then it will show video time line as well.

Reply:
done!